internal/query/instance.go

```diff
@@ -280,7 +289,8 @@ func prepareInstancesQuery(ctx context.Context, db prepareDatabase) (sq.SelectBu
 	return sq.Select(
 			InstanceColumnID.identifier(),
 			countColumn.identifier(),
-		).From(instanceTable.identifier()),
+		).From(instanceTable.identifier()).
```

Comment:
Intended that you added a join here and there is a "InstanceQuery", "InstancesQuery" and "InstanceDomainQuery", not a "InstancesDomainQuery"?

Reply:
I do not understand your comment. What should I do?

Reply:
As the InstanceQuery does not contain the domain-table I would opt to have it consistent to create a instancesDomainQuery instead of modifying InstancesQuery, but I think this will require some additional conditions somewhere.

so you would have some databasis for:
InstanceQuery -> InstancesQuery
InstanceDomainQuery -> InstancesDomainQuery

Reply:
Well, in the current query there is already a merge happening with the domains table. It's just that it happens in a sub-query and I didn't manage to unnest it without trouble. I don;t have enough knowledge about this query builder to figure that out.

IMO, the cleanest solution would be to not have a nested select with the merge and filter directly on the join.